src/components/shared/widget.js

```diff
@@ -57,8 +57,9 @@ const WidgetSelector = ({widget}) => {
             transcript={video?.relationships?.field_media_file?.publicUrl} /> 
             : null;
         case "paragraph__section":
+            let HeadingLevelSec = (widget.field_heading_level ? widget.field_heading_level : "h2");
```

Comment:
This is where you could add logic to check if heading_level is h5 or h6, and if so, assign it a value of "h4" instead